libsolidity/analysis/TypeChecker.cpp

```diff
@@ -613,6 +615,7 @@ bool TypeChecker::visit(VariableDeclaration const& _variable)
 
 void TypeChecker::visitManually(
 	ModifierInvocation const& _modifier,
+	ContractDefinition const* _contract,
```

Comment:
Is this different from `m_currentContract`?

Reply:
I found m_currentContract just later. so I dropped that part again.